lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -1,5 +1,7 @@
 import math
```

Comment:
the only place this package is used is in `PassiveCalibrationModel.calibration_results()` whereas in the `ActiveCalibrationModel` version of the same method you use numpy's implementation. My vote would be to get rid of this import and use numpy throughout

Reply:
Alright.